src/simply-rets-post-pages.php

```diff
@@ -27,6 +26,16 @@
 
 add_filter("rewrite_rules_array", array("SimplyRetsCustomPostPages", "srAddRewriteRules"));
 
+/**
+ * Don't run `single_template` filter on Block Themes
+ * A Block Theme is unable to find the correct template for
+ * sr-listings pages when this is enabled. Simply not running it
+ * allows the theme to choose the correct template.
+*/
+if (!wp_is_block_theme()) {
```

Comment:
TODO: We need to check if this function exists to prevent errors on older versions of WordPress.

```php
function_exists("wp_is_block_theme")
```